src/components/Words/Words.js

```diff
@@ -1,140 +0,0 @@
-import { useState, useEffect } from "react";
-
-import Card from "../UI/Card";
-import classes from "./Words.module.css";
-import wordsEn from "../../data/english";
-import wordsFr from "../../data/french";
-
-const Words = props => {
-  const [clickedCards, setClickedCards] = useState([]);
-  const [shuffledWordsEn, setShuffledWordsEn] = useState([]);
-  const [shuffledWordsFr, setShuffledWordsFr] = useState([]);
-
-  const clearedCardsObj = props.setClearedCards;
-
-  const shuffleArray = array => {
-    for (let i = array.length - 1; i > 0; i--) {
-      const j = Math.floor(Math.random() * (i + 1));
-      const temp = array[i];
-      array[i] = array[j];
-      array[j] = temp;
-    }
-    return array;
-  };
-
-  const translationMap = {
-    Hello: "Bonjour",
-    Cat: "Chat",
-    Strawberry: "Fraise",
-    Boy: "Garçon",
-    I: "Je",
-  };
-
-  const getKeyByValue = (object, value) => {
-    return Object.keys(object).find(key => object[key] === value);
-  };
-
-  const evaluate = props => {
-    const [first, second] = clickedCards;
-    const wordOne = document.getElementById(first);
-    const wordTwo = document.getElementById(second);
-
-    const backToDefault = () => {
-      wordOne.style.background = "white";
-      wordTwo.style.background = "white";
-    };
-
-    if (
-      translationMap[`${first}`] === second ||
-      getKeyByValue(translationMap, `${first}`) === second
-    ) {
-      clearedCardsObj(prev => ({
-        ...prev,
-        [first]: true,
-        [second]: true,
-      }));
-    } else if (
-      translationMap[`${first}`] !== second ||
-      getKeyByValue(translationMap, `${first}`) !== second
-    ) {
-      wordOne.style.background = "crimson";
-      wordTwo.style.background = "crimson";
-
-      setTimeout(backToDefault, 1000);
-    }
-  };
-
-  const cardClickHandler = e => {
-    const word = e.target.id;
-
-    const cardBackgroundColour = () => {
-      document.getElementById(word).style.background = "#dee6b4";
-    };
-
-    cardBackgroundColour();
-
-    if (clickedCards.length === 1) {
-      setClickedCards(prev => [...prev, word]);
-    } else {
-      setClickedCards([word]);
-    }
-  };
-
-  useEffect(() => {
```

Comment:
We could abstract these useEffect hooks into their own files and name them. Like `useEvaluate.js`, and then they'd be named custom hooks!